amplify.yml

```diff
@@ -33,4 +33,4 @@ test:
         - 'npx cypress run --reporter mochawesome --reporter-options "reportDir=cypress/report/mochawesome-report,overwrite=false,html=false,json=true,timestamp=mmddyyyy_HHMMss"'
     postTest:
       commands:
-        - npx mochawesome-merge cypress/report/mochawesome-report > cypress/report/mochawesome.json
+        - npx mochawesome-merge cypress/report/mochawesome-report/*.json > cypress/report/mochawesome.json
```

Comment:
```suggestion
        - npx mochawesome-merge@4 cypress/report/mochawesome-report/*.json > cypress/report/mochawesome.json
```

Reply:
@kevinold I think this is a better practice - specify at least the major version of the tool when using `npx` to avoid breaking change like https://github.com/Antontelesh/mochawesome-merge#migration-to-v4